pageserver/src/tenant/timeline.rs

```diff
@@ -2417,6 +2429,7 @@ impl Timeline {
                                 Err(e) => return Err(PageReconstructError::from(e)),
                             };
                             cont_lsn = lsn_floor;
+                            *read_count += 1;
```

Comment:
Does it count as a `fs_layer` if it was cached? I'm not very familiar with the caching logic of indexes and layers but I think there is some.

Reply:
It counts for now. We can have a separate metrics to count actual I/O in one get page request (i.e., include fetching index and fetching data) later.

Reply:
Whatever is useful for your measurements for now, let's just make it clear what we're measuring